Gems/AtomLyIntegration/AtomViewportDisplayIcons/Code/Source/AtomViewportDisplayIconsSystemComponent.cpp

```diff
@@ -354,20 +361,22 @@ namespace AZ::Render
     {
         // Once the shader is loaded, register it with the dynamic draw context
         Data::Asset<RPI::ShaderAsset> shaderAsset = asset;
-        AtomBridge::PerViewportDynamicDraw::Get()->RegisterDynamicDrawContext(m_drawContextName, [shaderAsset](RPI::Ptr<RPI::DynamicDrawContext> drawContext)
+        AtomBridge::PerViewportDynamicDraw::Get()->RegisterDynamicDrawContext(m_drawContextName, [shaderAsset](RPI::Ptr<RPI::DynamicDrawContext> dynamicDraw)
             {
                 AZ_Assert(shaderAsset->IsReady(), "Attempting to register the AtomViewportDisplayIconsSystemComponent"
                     " dynamic draw context before the shader asset is loaded. The shader should be loaded first"
                     " to avoid a blocking asset load and potential deadlock, since the DynamicDrawContext lambda"
                     " will be executed during scene processing and there may be multiple scenes executing in parallel.");
 
                 Data::Instance<RPI::Shader> shader = RPI::Shader::FindOrCreate(shaderAsset);
-                drawContext->InitShader(shader);
-                drawContext->InitVertexFormat(
-                    { {"POSITION", RHI::Format::R32G32B32_FLOAT},
-                     {"COLOR", RHI::Format::R8G8B8A8_UNORM},
-                     {"TEXCOORD", RHI::Format::R32G32_FLOAT} });
-                drawContext->EndInit();
+                dynamicDraw->InitShader(shader);
+                dynamicDraw->InitVertexFormat({ { "POSITION", RHI::Format::R32G32B32_FLOAT },
+                                                { "COLOR", RHI::Format::R8G8B8A8_UNORM },
+                                                { "TEXCOORD", RHI::Format::R32G32_FLOAT } });
```

Comment:
nit: I preferred the old style as it has shorter lines.

Reply:
This is clang-format, our new overlord; I think the benefits of autoformat outweigh the visual parsing issues, but there's definitely a bit of friction because most of the codebase isn't formatted. In this case I'd say feel free to format and check-in.

Reply:
Yeah this is just clang-format, any lines I change I run it on to slowly update things to the new style. There's a conversation happening on Discord at the moment with @amznestebanpapp and @nemerle about enabling auto-formatting for PRs so we can stop worrying about formatting differences

Reply:
I've tagged you both in the thread as I wasn't sure how to link it here